documentation/book/con-customizing-pod-disruption-budgets.adoc

```diff
@@ -0,0 +1,33 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-customizing-deployments.adoc
+
+[id='con-customizing-pod-disruption-budgets-{context}']
+= Customizing Pod Disruption Budgets
+
+{ProductName} automatically creates a Pod Disruption Budget for every `StatefulSet` or `Deployment` which it creates.
+By default, the Pod Disruption Budgets always allow up to one Pod to be unavailable at a time by setting the `maxUnavailable` value in `PodDisruptionBudget.spec` to `1`.
+You can change this default setting using the Pod Disruption Budget template.
+The Pod Disruption Budget template is effective on each type of cluster (Kafka and Zookeeper; Kafka Connect and Kafka Connect with S2I support; and Kafka Mirror Maker).
+
+The following example shows customized Pod Disruption Budget fields on a `template` property:
+
+[source,yaml,subs=attributes+]
+----
+# ...
+template:
+    podDisruptionBudget:
+        metadata:
+            labels:
+                key1: label1
+                key2: label2
+            annotations:
+                key1: label1
+                key2: label2
+        maxUnavailable: 1
+# ...
+----
+
+.Additional resources
+
+* For more information, see xref:type-PodDisruptionBudgetTemplate-reference[].
```

Comment:
* The [Managing Pods](https://access.redhat.com/documentation/en-us/openshift_container_platform/3.6/html/cluster_administration/admin-guide-manage-pods) chapter of the OpenShift Container Platform documentation.

Reply:
As this is upstream documentation, we should use links to Kubernetes. We should change it OpenShift link only in the downstream repo. I used here the link which you asked about in another comment.